docs/fundamentals/client/client.md

```diff
@@ -668,7 +663,26 @@ with Flow().add() as f, open('output.txt', 'w') as fp:
     )
 ```
 
+````{admonition} What errors can be handled by the callback
+:class: caution
+Callback can handle errors that are caused by Executors raising an Exception.
+It will not receive exceptions or from the Gateway having connectivity errors with the Executors or between the Client and the Gateway.
```

Comment:
I don't understand this sentence :/

Reply:
that "networking" exceptions are not passed to the `on_error` callback, how could I better phrase this?